jinja2/react_base.html

```diff
@@ -95,10 +95,8 @@
   {% block pageheader %}{% endblock %}
 
   <!-- Content will go here -->
-  <main id="content">
```

Comment:
The a11y nav links to `#content`, so I think we should still keep the ID (whether it be here or one of the children) or update the link to point at a new ID.

Reply:
Doh, good catch. Will update.